src/components/Onboarding/OnboardingBuzz.tsx

```diff
@@ -149,7 +149,7 @@ export function OnboardingBuzz() {
               Done
             </Button>
           </Group>
-          <RecaptchaNotice />
+          {/* <RecaptchaNotice /> */}
```

Comment:
We likely still wanna have this for legal reasons I believe. We may need to update the copy.